### DISCUSSION TITLE
Welcome to .NET Interactive Discussions!

### DISCUSSION THREAD

**Discussion by jonsequitur** [dotnet/interactive#1280] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by khoanguyen123** [comment]
Is the .NET logging framework supported? The following doesn't display anything:

```C#
using Microsoft.Extensions.Logging;

var loggerFactory = LoggerFactory.Create(builder => {
                builder
                .ClearProviders()
                .SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Debug)
                .AddConsole();
            });

var logger= loggerFactory.CreateLogger<Object>();
logger.LogInformation("It should display but doesn't. Why?");
